### PR TITLE
[nginx-ingress-controller]: Improve performance (listen backlog=net.core.somaxconn)

### PR DESCRIPTION
**Pull request by aledbf**

### CODE REVIEW

**Comment by aledbf** [comment]
ping @bprashanth

File: ingress/controllers/nginx/nginx/utils.go [hunk 1]
```diff
@@ -220,3 +221,14 @@ func diff(b1, b2 []byte) (data []byte, err error) {
 	}
 	return
 }
+
+// sysctlSomaxconn returns the value of net.core.somaxconn, i.e.
+// maximum number of connections that can be queued for acceptance
+func sysctlSomaxconn() int {
+	maxConns, err := sysctl.GetSysctl("net.core.somaxconn")
+	if err != nil || maxConns < 512 {
```
Comment (bprashanth) [inline ingress/controllers/nginx/nginx/utils.go]: please log it too

**Comment by bprashanth** [comment]
LGTM just logging nit

File: ingress/controllers/nginx/nginx/utils.go (hunk 1 above)
Comment (aledbf) [inline ingress/controllers/nginx/nginx/utils.go]: done

**Comment by bprashanth** [comment]
LGTM thanks

**Comment by foolusion** [comment]
Hi i am wondering why this wasn't added to port 80 and 443 as well? @aledbf @bprashanth

**Pull request merged**